pkg/operator/ceph/cluster/cleanup.go

```diff
@@ -201,6 +203,10 @@ func (c *ClusterController) getCephHosts(namespace string) ([]string, error) {
 			return hostNameList, errors.Wrapf(err, "could not list the %q pods", app)
 		}
 		for _, cephPod := range podList.Items {
+			if cephPod.Status.Phase != v1.PodRunning {
```

Comment:
Since the purpose of this loop is to get the node assignment of the pod, how about if we instead just check `if cephPod.Spec.NodeName != ""`? There could be other pod states besides `running` that have the node assignment such as `terminating`.

Reply:
make more sense. Updated the code to just check for empty node name.